src/css/index.css

```diff
@@ -1,27 +1,109 @@
+.container{
+  margin: 0 !important;
+}
+
 h1 {
   color: aquamarine;
 }
 
 #featured-work {
+  min-height: 90vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+#featured-work_images-container {
+  width: 100%;
   display: flex;
   flex-direction: row;
-  padding: 30px 0;
+  justify-content: space-evenly;
+  align-items: center;
 }
 
 .featured-img {
   margin: 0 10px;
+  border-radius: 15px;
+}
+
+.seperator {
+  border-top: 1.5px solid var(--button-green);
+  width: 30%;
+  margin: 60px auto;
+}
+
+#testimonials-container {
+  margin: 40px 10%;
+  padding: 1px 0;
+  display: grid;
+  grid-template-columns: repeat(12,  1fr);
+  grid-template-rows: repeat(1,1fr);
+  column-gap: 2rem;
+  row-gap: 2rem;
+}
+
+#testimonials-container > div:nth-child(1) {
+  grid-column: 1 / span 3;
+  grid-row: 2 / 3;
+  font-size: x-small;
+}
+
+#testimonials-container > div:nth-child(4) {
+  grid-column: 9 / span 4;
+  grid-row: 2 / 4;
+  font-size: medium;
+}
+
+
+#testimonials-container > div:nth-child(3) {
+  grid-column: 2 / span 4;
+  grid-row: 3 / 5;
+  font-size: medium;
+}
+
+#testimonials-container > div:nth-child(2) {
+  grid-column: 5 / span 4;
+  grid-row: 1 / 2;
+  font-size: small;
+}
+
+#testimonials-container > div:nth-child(5) {
+  grid-column: 7 / span 3;
+  grid-row: 4 / 5;
+  font-size: x-small;
+}
+.wrapper {
+  background-color: var(--background-color_2);
+  padding: 40px;
 }
 
 #about-container {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
+  max-width: 900px;
+  margin: 10px auto;
+  background-color: var(--background-color_2);
 }
 
 #about {
   display: flex;
   flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+#about > img {
+  margin: 0 30px;
 }
 #home-about {
   border-radius: 50%;
 }
 
+#call-to-action, #call-to-action>button {
+  margin: 0 auto;
```

Comment:
Please make sure that the text has a proper margin to the CTA itself when using on small devices

Reply:
I haven't addressed any responsiveness.  I will make that a separate issue